src/init.hpp

```diff
@@ -112,7 +113,7 @@ namespace gpudrive
         uint32_t maxNumControlledAgents = 10000;                               // Arbitrary high number to by default control all vehicles
         bool IgnoreNonVehicles = false;                                        // Default: false
         FindRoadObservationsWith roadObservationAlgorithm{
-            FindRoadObservationsWith::KNearestEntitiesWithRadiusFiltering};
+            FindRoadObservationsWith::AllEntitiesWithRadiusFiltering};
```

Comment:
probably do not want this change to do this?

Reply:
Yeah I think this branch contains a lot of changes we do not want to include. I am starting a different branch with only the minimal required changes to access the z logs